include/glow/Graph/Graph.h

```diff
@@ -549,6 +549,10 @@ Function *differentiate(Function *F, const TrainingConfig &config,
 /// gradients that flow into V.
 Variable *generateRecordGradientNode(Function &G, Variable *V);
 
+/// Helper vectors for common transpose shuffles.
+const auto NCHW2NHWC = {0u, 2u, 3u, 1u};
+const auto NHWC2NCHW = {0u, 3u, 1u, 2u};
```

Comment:
Is it safe to put constant definitions into a header file? May be it should be macro-definitions or constexprs or some kind of declarations that are initialized elsewhere in a CPP file?